src/test/kotlin/no/nav/syfo/producer/arbeidsgivernotifikasjon/ArbeidsgiverNotifikasjonProdusentSpek.kt

```diff
@@ -46,11 +46,11 @@ class ArbeidsgiverNotifikasjonProdusentSpek : DescribeSpec({
 
     describe("ArbeidsgiverNotifikasjonProdusentSpek") {
         it("Should send oppgave") {
-            runBlocking { arbeidsgiverNotifikasjonProdusent.createNewTaskForArbeidsgiver(arbeidsgiverNotifikasjon) }
+            launch { arbeidsgiverNotifikasjonProdusent.createNewTaskForArbeidsgiver(arbeidsgiverNotifikasjon) }
```

Comment:
Det ser ut som vi kan droppe `launch`/`runBlocking` her. Antakeligvis er det støtte for det ut av boksen fra kotest.

Reply:
Det har du rett i! Endrer